web/src/routes/(user)/albums/[albumId]/+page.svelte

```diff
@@ -142,6 +149,13 @@
     }
   });
 
+  const autoGrowHeight = () => {
+    // little hack so that the height of the text area is correctly initialized
+    textarea.scrollHeight;
+    textarea.style.height = '5px';
+    textarea.style.height = `${textarea.scrollHeight}px`;
+  };
```

Comment:
It looks like you just copied and pasted all of this stuff. Can you re-implement this without copy/pasting please?

Reply:
What do you want me to change ? 
Just put it in a `.ts` file ?

Reply:
Maybe look into a way to:
- add a directive/attribute/use:autogrow thing that will just do it automatically
- add a custom `TextArea` component that we can use everywhere we want a text box

also what is line 154?

Reply:
Without it, the textarea height is not correctly initialized:

https://github.com/immich-app/immich/assets/74269598/9ca34051-ddc9-4b5c-bc81-2f46b551a253